tests/containers/rootless_podman.pm

```diff
@@ -38,11 +38,8 @@ sub run {
 
     # add testuser to systemd-journal group to allow non-root
     # user to access container logs via journald event driver
-    # to avoid flakes w/ Podman <=4.0.0
-    $podman_version = get_podman_version();
-    if (package_version_cmp($podman_version, '4.0.0') <= 0) {
-        assert_script_run "usermod -a -G systemd-journal $testapi::username";
-    }
+    # bsc#1207673, bsc#1218023
+    assert_script_run "usermod -a -G systemd-journal $testapi::username";
```

Comment:
iirc this is not required on openSUSE, so it shouldn't be run there

Reply:
When it's not required on openSUSE, do you mean Tumbleweed? Isn't it then again podman version specific?

Reply:
Afaik this is related to the journald config that is different on SLE and openSUSE, but I don't know about the settings on Leap. Nevertheless, this is unrelated to podman, but is a systemd config.

Reply:
Just saying that also openSUSE Tumbleweed is a moving target, and those tests also run on Leap, so they need to incorporate that as well.

Reply:
Leap should have the same logging setup as sles. TW uses journald logging by default

Reply:
@dcermak is right, historically, SLE has had volatile storage for journald for user journals and access control doesn't seem to work in such a scenario and that's perhaps the reason why we see this issue. Default storage for journald is `auto` but it falls back to `volatile` if `/var/log/journal` isn't present which is the case on SLE but not on TW. I'd believe Leap would be similar to SLE but for now, I'm gating this check to only run on non-tumbleweed.

cc/ @Vogtinator

Reply:
This is not ideal, then `usermod -a -G systemd-journal $testapi::username` is executed by default on all products. Especially with ALP and a set of newer products we don't want to apply a workaround as default.

It's better to apply the workaround explicitly on all flavors where the workaround is expected to happen, also if this makes this `if` statement much more verbose - but otherwise we loose coverage of testing the system without the workaround on newer products, because nobody will remember to adjust this test code afterwards.

Workarounds need to be explicitly included.

Reply:
> Especially with ALP and a set of newer products we don't want to apply a workaround as default.

Trying to understand why would this be a problem. Besides, I don't know whether to call this simply a workaround because this is one of the suggested *solutions* from upstream for people running into issues with rootless logging. But then again, I agree with you on being explicit here, how about we do this only for SLE and Leap? `(is_leap("<16.0") || is_sle("<16"))`Agreed, how about then we do this only for SLE and Leap? `(is_leap("<16.0") || is_sle("<16"))`

Reply:
> `(is_leap("<16.0") || is_sle("<16"))`

yep. Not sure whether SLEM is also affected.

Reply:
SLEM tests don't seem to be affected by this.